domain/src/main/kotlin/no/nav/su/se/bakover/domain/Sak.kt

```diff
@@ -49,7 +50,29 @@ data class Sak(
     val utbetalinger: List<Utbetaling>,
     val revurderinger: List<Revurdering> = emptyList(),
     val vedtakListe: List<Vedtak> = emptyList(),
-)
+) {
+    fun hentÅpneRevurderinger(): List<Revurdering> {
+        return revurderinger.filterNot {
+            it is IverksattRevurdering
+        }
+    }
+
+    fun hentÅpneSøknadsbehandlinger(): List<Søknadsbehandling> {
+        return behandlinger.filterNot {
+            it is Søknadsbehandling.Iverksatt
+        }
+    }
+
+    fun hentÅpneSøknader(): List<Søknad> {
+        val ikkeLukkedeSøknader = søknader.filterNot {
+            it is Søknad.Lukket
+        }
+        val søknaderMedSøknadsbehandling = behandlinger.map {
+            it.søknad
+        }
+        return ikkeLukkedeSøknader.minus(søknaderMedSøknadsbehandling)
+    }
+}
```

Comment:
Litt generelt for alle disse, er "åpen" et begrep vi vanligvis bruker? Kanskje? Kunne evt vurdert noe i baner av "ubehandlede søknader" og "revurdering til behandling", eller noe i den duren.